best.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="best.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <ul>
+                <li class="logo"><a href="#">MY LOGO</a></li>
+                <li class="ham"><a href="#"><img src="images/Union1.png" alt="union"></a></li>
+            </ul>
+        </header>
+
+        <section>
+            <h1 class="headingone">I'm Alfred <br> Glad to see <br> you!</h1>
+            <p class="paragraph">I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact.</p>
+            <p class="lets_connect">LET'S CONNECT</p>
+            <div class="social">
+                <ul>
+                    <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+                    <li><a href="#"><img src="images/linkedin.png" alt="linked in"></a></li>
+                    <li><a href="#"><img src="images/mid.png" alt="mid"></a></li>
+                    <li><a href="#"><img src="images/git.svg" alt="git"></a></li>
+                    <li><a href="#"><img src="images/fin.svg" alt="fing"></a></li>
+                </ul>
+            </div>
+        </section>
+        <button type="button">See Projects</button>
+        <section>
+            <H2 class="headingone">About <br>Myself</h2>
+            <p class="paragraph">I
+                Hello I’m a software developer!
+                I can help you build a product ,
+                feature or website Look through some of
+                my work and experience! If you like what
+                you see and have a project you need coded,
+                don’t hestiate to contact me.
+            <p class="lets_connect">LET'S CONNECT</p>
+            <div class="social">
+                <ul class="icons">
+                    <li><img src="images/twitter.png" alt="twitter"></li>
+                    <li><img src="images/linkedin.png" alt="linked in"></li>
+                    <li><img src="images/mid.png" alt="mid"></li>
+                    <li><img src="images/git.svg" alt="git"></li>
+                    <li><img src="images/fin.svg" alt="fing"></li>
+                </ul>
+            </div>
+            <button type="button">Get my resume</button>
+            <section class="lang">
+                <p class="section1"> Languages </p>
+                <p><img src="images/Union.png" alt="drop down"></p>
+            </section>
+            <ul class="languages">
+                <li class="l-items"> <a href="#"><img src="images/js.png" alt="java"><span
```

Comment:
- [ ]  As part of the main requirements, please add a project section to your portfolio. The project section is an essential part of a portfolio. So kindly implement the suggestion given by the previous reviewer.
- [ ] The middle section of the project is missing so kindly implement it.
## your design
![his](https://user-images.githubusercontent.com/81727988/165733112-709653e7-d808-449e-b502-2033822950c3.png)
## Figma design
![figma](https://user-images.githubusercontent.com/81727988/165733151-85966b52-5a72-4c6e-a959-a7cf84a585dd.png)

Reply:
@roshan-bajgain 
should I include the contact section?
Once I finish my work is it possible to communicate with you promptly so that I don't utilize all my reviews, or if I have changes I make them ASAP?. I have used three reviews so far.